homework/index.js

```diff
@@ -30,14 +30,93 @@
     return elem;
   }
 
+  function createOption(repositories, select) {
+    createAndAppend('option', select, { text: 'Select a repository', disabled: 'disabled' });
+    repositories
+      .sort((a, b) => a.name.localeCompare(b.name))
+      .forEach(repository => {
+        createAndAppend('option', select, { text: repository.name, value: repository.name });
+      });
+  }
+
+  function renderRepoBasisInfo(repository, ul) {
+    const url = `https://api.github.com/repos/HackYourFuture/${repository}`;
+    const container = document.getElementById('root');
+    fetchJSON(url, (err, repo) => {
+      if (err) {
+        createAndAppend('div', container, { text: err.message, class: 'alert-error' });
+      }
```

Comment:
In case of an error, you render the error to the page but then stumble into code you should be executing. What about adding a `return` statement?

But, as I mentioned, we do not need this fetch at all.